File: frontend/staff-web-app/src/models/leave/UpdateLeaveRequest.js
export default class UpdateLeaveRequest {
    leaveId;
    applicant;
    fromDateTime;
    toDateTime;

    constructor(leaveId, applicant, fromDateTime, toDateTime) {
        this.leaveId = leaveId;
        this.applicant = applicant;
        this.fromDateTime = fromDateTime;
        this.toDateTime = toDateTime;
    }
}

File: backend/src/main/java/capstone/rt04/retailbackend/request/leave/UpdateLeaveRequest.java
package capstone.rt04.retailbackend.request.leave;
import capstone.rt04.retailbackend.entities.Staff;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import java.time.LocalDate;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UpdateLeaveRequest {
    private Long leaveId;
    private Staff applicant;
    private LocalDate fromDateTime;
    private LocalDate toDateTime;
}
